### DISCUSSION TITLE
Welcome to XMOJ-Script Discussions!

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PythonSmall-Q** [comment]
Hey, Guys, I am PythonSmall-Q also known as Sean, Boomzero and I are both Developers of this Script

**Comment by zsTree0830** [comment]
Hi, I'm zsTree, and I'm glad to be the developer of this script.

**Reply by boomzero** [reply]
Maybe join our Wechat group?

**Reply by zsTree0830** [reply]
> Maybe join our Wechat group?

那把我再拉进去吧（当时退了原因给cln说过）

**Reply by boomzero** [reply]
Done
